cli/src/jobs_runner/jobs_resolver.rs

```diff
@@ -146,9 +158,13 @@ fn is_job_involved(target: Target, current_job: JobType, main_job: &JobType) ->
 
             // Tests for TS and WASM targets inquire that those targets are built
             JobType::Test { .. } => match target {
-                // Running tests for rust jobs doesn't inquire running build on them.
+                // * Running tests for rust jobs doesn't inquire running build on them.
+                // * They should excluded in the filter if it's not included in the original targets
+                //   before being flatted. This is to avoid running test on core if we want to run
+                //   test on Binding or Wrapper.
                 Target::Core | Target::Cli | Target::Updater => {
                     matches!(current_job, JobType::Test { .. })
+                        && original_targets.contains(&target)
```

Comment:
A generic job property could be used here.

Reply:
Same as above. It would be better to make it with adding configuration files issue